frame/bridge/relayer-game/src/lib.rs

```diff
@@ -20,6 +20,30 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+pub mod migration {
+	// --- darwinia-network ---
+	use crate::*;
+
+	pub fn migrate<T, I>()
+	where
+		T: Config<I>,
+		I: Instance,
+	{
+		<RelayHeaderParcelToResolve<T, I>>::put(<Vec<RelayHeaderId<T, I>>>::new());
```

Comment:
why not `remove_all()` here?

Reply:
This is a value, not a map. 

I think you mean: `<RelayHeaderParcelToResolve<T, I>>::kill()`.

But it's okay to put an empty `Vec` into it.